.gitignore

```diff
@@ -1,6 +1,7 @@
 .gradle/
 .classpath
 .project
+.vscode
```

Comment:
Oh, I meant that this would go in your own personal `$HOME/.gitignore` rather than the project's gitignore. Otherwise, we will have to update the project each time a contributor uses a different editor/IDE.

Reply:
You might also need to update your `$HOME/.gitconfig` to use a personal global gitignore.

```gitconfig
[user]
    email = aazores@redhat.com
    name = Andrew Azores
...
[core]
    excludesfile = ~/.gitignore
...
```

Reply:
git config --global core.excludesFile ~/.gitignore .. i did that but the way you did it looks cleaner

Reply:
That command probably just edits the `~/.gitconfig` the same way that I did it manually.

Reply:
yes, the config file now looks like what you did